shared/actions/config/index.js

```diff
@@ -69,6 +69,7 @@ export function bootstrap (): AsyncAction {
     } else {
       Promise.all(
         [dispatch(getCurrentStatus()), dispatch(getExtendedStatus()), dispatch(getConfig())]).then(() => {
+          dispatch({type: Constants.bootstrapped, payload: null})
```

Comment:
when we bootstrap we dispatch this action so we can count it. really this would be a great use-case for sagas. instead of having to know to nav based on this and emit this counter increment those would be side effects of bootstrapping